.github/workflows/issue-assigner.yml

```diff
@@ -0,0 +1,12 @@
+name: 'Issue assigner'
+on:
+  issues:
+    types: [opened]
+jobs:
+  assign:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: patrickhulce/issue-assigner@eeec7a1
+        with:
+          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+          maintainers: 'patrickhulce,paulirish,connorjclark,adamraine'
```

Comment:
if anyone else would like to be a part of this, happy to add :) 

@adrianaixba @Beytoven perhaps?

Reply:
I'm happy to take issues too

Reply:
```suggestion
          maintainers: 'patrickhulce,paulirish,connorjclark,adamraine,brendankenny'
```

Reply:
You got it :)